addons/website/models/ir_attachment.py

```diff
@@ -15,7 +15,7 @@ class Attachment(models.Model):
     # related for backward compatibility with saas-6
     website_url = fields.Char(string="Website URL", related='local_url', deprecated=True, readonly=False)
     key = fields.Char(help='Technical field used to resolve multiple attachments in a multi-website environment.')
-    website_id = fields.Many2one('website')
+    website_id = fields.Many2one('website', ondelete='cascade')
```

Comment:
noo falback make it crappy

Need to override unlink:
key & website_Id = website
or .custom.... scss/js

Reply:
Indeed, we don't want to unlink a SO pdf (that will only have its website_id flag if the website's domain is set and match the current URL ('bug' on get_current_website fallback))
Done